test/src/loaders/ncbi_downloader/ncbi_downloader_test.py

```diff
@@ -1,5 +1,103 @@
+import os
```

Comment:
The tests present look good, but given tests aren't required in prototype mode, I'm not commenting on uncovered code or missing tests

Reply:
👍 yup. there are uncovered codes. I mainly do this to help me check some methods are working and returning properly.